pom.xml

```diff
@@ -372,4 +373,32 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
```

Comment:
This plugin package the dependencies into the jar, we need this only for the layer so its behind a profile and we activate it on the layer build